CodePush.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name                = 'CodePush'
-  s.version             = '1.10.1-beta'
```

Comment:
Since the `s.source` property specifies the Git tag using the `s.version` property, would it cause any issues for folks if we removed the `-beta` suffix here since the actual Git tag would include it?